packages/fxa-content-server/app/scripts/views/behaviors/settings.js

```diff
@@ -26,7 +26,7 @@ export default function(defaultBehavior, options = {}) {
         let success = t('Account verified successfully');
         let endpoint = 'settings';
 
-        if (options.success) {
+        if (options.success !== null) {
```

Comment:
Strict comparison because `undefined == null`, but `undefined !== null`, and we want to be able to intentionally unset this value via `options.success`.